addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -163,7 +163,7 @@ $o-we-sidebar-content-field-colorpicker-dropdown-active-color: $o-we-fg-lighter
 $o-we-sidebar-content-field-colorpicker-cc-width: 208px !default;
 $o-we-sidebar-content-field-colorpicker-cc-height: 26px !default;
 
-$o-we-sidebar-content-field-input-max-width: 60px !default;
+$o-we-sidebar-content-field-input-max-width: 75px !default;
```

Comment:
What about keeping 60px, renaming the variable and using it as a min-width, won't that work and increase the size of the input if the unit text is longer?